context/app/static/js/components/detailPage/visualization/VisualizationNotebookButton/VisualizationNotebookButton.tsx

```diff
@@ -52,7 +52,7 @@ function VisualizationNotebookButton({ uuid, hubmap_id, mapped_data_access_level
     },
     {
       children: 'Download Jupyter Notebook',
-      onClick: downloadNotebook,
+      onClick: downloadNotebook(),
```

Comment:
```suggestion
      onClick: downloadNotebook,
```
It should work without this if you just delete the `return () => {` and its closing brace in `downloadNotebook`.

Reply:
Thanks!